---
layout: page
title: Research Projects
subtitle: Past and present research projects
---
## Data Engineering and Multiomic Integration at Novonesis
#### *Postdoctoral Scientist - Bioinformatics (Data Engineer I), Novonesis, 2022–Present*
At Novonesis, I led efforts to optimize data flow pipelines, integrate multiomic datasets, and standardize workflows for next-generation sequencing (NGS) data analysis, driving innovation and operational efficiency.

#### Key contributions:

- **Workflow Automation**: Refactored pipeline code to adopt an object-oriented programming approach, replacing legacy data table manipulations for greater modularity and scalability.
- **Streamlined Processes**: Consolidated six Jupyter Notebooks into a version-controlled Python workflow, significantly enhancing reproducibility and reducing workflow fragmentation.
- **Pain Point Elimination**: Automated previously manual data entry processes, alleviating key bottlenecks for the sequencing team and saving over 600 hours annually.
- **Data Integration and Insights**: Established a SQL-based matedata capture system integrated with Benchling, enabling longitudinal tracking of sequencing sample data and providing capabilities for trend analysis and historical comparisons that were previously unavailable.
- **Multiomic Data Integration**: Designed and implemented workflows to integrate genomics, transcriptomics, proteomics, and metabolomics datasets, delivering actionable systems-level insights for downstream analyses.
- **SOP Development**: Authored standardized operating procedures (SOPs) for proteomics, transcriptomics, and metabolomics workflows to ensure consistency, scalability, and adherence to industry best practices.

---

## **Bacteriophage and Antimicrobial Resistance in Poultry Environments**  
#### *Dissertation Work, Eisen Lab, UC Davis, 2019–2022*  
This research investigated microbial relationships in agricultural environments, focusing on the dissemination of antimicrobial resistance (AMR) and the ecological roles of bacteriophage and bacteria.  

#### Key findings and contributions:  
- Identified that bacteriophage significantly associated with the retention and dissemination of AMR genes in poultry-associated environments, serving as potential reservoirs of resistance.  
- Demonstrated correlations between bacteriophage-encoded auxiliary genes and AMR genes, shedding light on molecular mechanisms that enhance microbial survival in agricultural settings.  
- Showed that antibiotic usage in backyard poultry production directly impacts AMR gene prevalence and associated microbial dynamics.  
- Highlighted implications for the development of bacteriophage-based therapeutics, paving the way for innovative solutions to mitigate AMR dissemination.  

---

### **Characterization of Environmental Resistance Genes in NASA JPL Bacterial Isolates**  
#### *Dissertation Work, Eisen Lab, UC Davis, 2021–2023*  
This project analyzed whole-genome sequences from archival bacterial isolates collected from spacecraft assembly facilities to understand microbial survival in sterile, extreme environments.  

#### Key findings and contributions:  
- Identified resistance genes that contribute to bacterial survival under sterilization practices, including efflux pumps and stress response genes.  
- Observed shifts in resistance gene profiles over time within taxonomic groups, suggesting adaptive evolution to persistent selective pressures in sterile environments.  
- Found significant differences in resistance gene profiles between bacterial taxa, reflecting ecological specialization.  
- Provided insights into biological processes impacted by these resistance genes, including DNA repair, membrane integrity, and oxidative stress responses, advancing understanding of microbial resilience in extreme environments.

---

## Genomic assembly and characterization of attenuated *Streptococcus iniae*
#### *Collaborative project, UC Davis, 2021*
Hybrid assembly and refinement using Oxford Nanopore and Illumina sequencing to resolve assembly of attenuated bacterial candidate for aquaculture vaccine.

---

## *Salmonella* strain-level recovery and identification 
#### *Bioinformatic consulting, 2020 - 2021*
Advised on bioinformatic techniques and experimental design for quantification of Salmonella strains from retail poultry products.

---

## Amplicon analysis of fungal communities
####  *Bioinformatic consulting, 2020*
Advised and implemented bioinformatic pipeline reporting predicted functional and diversity metrics of previously generated ITS amplicon sequencing.

---

## Microbial Communities of Rehabilitated Soils in the Brazilian Cerrado
#### *Side project, UC Davis, 2018-2021*

16S rRNA gene analysis to understand how mining in the Brazilian Cerrado impacted bacterial communities in affected soils, and how rehabilitation through the application of rehabilitated biosolids impacted the microbial dynamics. The manuscript of this project is in preparation with anticipated submission of July 2021. 

---

## Quanitification of Microbially-Derived Biotech Product Application Impact on Ruminal Functional and Quality Measurements
#### *R&D Internship Project, Alltech, 2017*
- Determined extent of biotech product application’s association with gene expression changes associated with tenderness, flavor, and juiciness in Angus steers. 
- Established baseline range of microbial degradation of volatile fatty acids in ruminal fluid of lactating dairy cattle.

---

## Superinfection Potential of Genetically Modified Nudivirus
#### *Assistant Research Associate work and Undergraduate Thesis research, Paratechs and Lepidext, 2014 - 2017*
My undergraduate capstone research project for my degree in [Agricultural and Medical Biotechnology at the University of Kentucky](https://www.uky.edu/academics/bachelors/agricultural-and-medical-biotechnology) was performed at [Lepidext](https://www.lepidext.com/) (a derivative of work at [ParaTechs Corporation](https://paratechs.com/)) in Lexington, Kentucky. This project aimed to develop a genetically modified nudivirus as a means of biological control of the *Helicoverpa zea* corn earworm pest, which is responsible for millions of dollars in agriculural damage each year. My independent project within the overall research was to determine the interactions between the wild-type nudivirus that exists ubiquitously in nature, and the genetically modified virus we developed. My work on this project led to the 2016 Global prize at the [Alltech Young Scientist competition](https://www.alltech.com/education/alltech-young-scientist), the largest international agri-science compeition.  

